src/user/login.go

```diff
@@ -70,13 +70,18 @@ func UserLogin(w http.ResponseWriter, req *http.Request) {
 			return
 		} else {
 			err2 := bcrypt.CompareHashAndPassword([]byte(user.Password), []byte(password))
-	
 			if err2 != nil {
 				utils.Error("UserLogin: Encryption error", err2)
 				utils.HTTPError(w, "User Logging Error", http.StatusUnauthorized, "UL001")
 				return
 			}
 
+			if utils.IsEmailEnabled() && utils.IsNotifyLoginEmailEnabled() && user.Email != "" {
+				if err := SendLoginNotificationEmail(user.Nickname, user.Email); err != nil {
+					utils.Error("UserLogin: Error while sending login notification email", err)
```

Comment:
Could you make this a MajorError instead of Error (it means admin will get a notif for the error)
I know other emails are not Major, but it's because MajorErrors were introduced much later